packages/local_auth/example/lib/main.dart

```diff
@@ -21,11 +21,21 @@ class MyApp extends StatefulWidget {
 
 class _MyAppState extends State<MyApp> {
   final LocalAuthentication auth = LocalAuthentication();
-  late bool _canCheckBiometrics;
-  late List<BiometricType> _availableBiometrics;
+
+  bool? _isSupported;
```

Comment:
Prefer  initializing variables rather than leaving them nullable.

Reply:
I'll add an enum to avoid the `null`

Reply:
Updated to use the `enum` `_SupportState` instead of a nullable `bool`